ui/package.json

```diff
@@ -104,5 +105,9 @@
     "paths": [
       "lib/bulma"
     ]
+  },
+  "dependencies": {
+    "columnify": "^1.5.4",
+    "yargs-parser": "^10.0.0"
```

Comment:
I've been adding everything to `devDependencies` to follow what ember does, maybe check with matt, in the grand scheme of things its no biggie.

Reply:
Oh yep, not sure how / why these got added.